content/docs/release-notes/archive/22-05-0/_index.md

```diff
@@ -0,0 +1,98 @@
+* **Atom**
+  * New Debug Rendering level component: There's a new level component that allows users to visualize and debug various aspects of the rendering. It can display normals, render only the diffuse or specular lighting, override specific properties for all materials in the scene, and more. 
+  * Pipeline Global Pass attachment reference: Users writing `.pass` files for rendering can now declare and reference attachments in a way that is global to the pipeline.
+  * Usability and performance improvements for DiffuseProbeGrid:
+    * DiffuseProbeGrid visualization: Displays individual probes and their raw irradiance values as spheres in the appropriate position in the level.
+    * Scrolling DiffuseProbeGrid:  Allows a DiffuseProbeGrid to be attached to the camera to apply diffuse GI wherever the camera is located.
+    * DiffuseProbeGrid `NumberOfRaysPerProbe` setting: The number of rays cast by each probe in the grid can now be adjusted in the DiffuseProbeGrid properties.  This value can be set differently per grid.
+    * DiffuseProbeGrid Irradiance query: Query the diffuse GI at any position and direction in the level. Improved DiffuseProbeGrid blending when multiple grids overlap and around the edges of a grid to blend with the global IBL cubemap.
+  * New **CubeMapCapture** component: Captures a diffuse or specular convolved cubemap at any location in the level.
+  * The MSAA state can now be changed in the `MainRenderPipeline.azasset` file by setting the `MultisampleState/samples` parameter.  Setting the samples to `1` will disable MSAA.
+  * Several improvements to Diffuse GI and ReflectionProbe visual quality.
+* **White box**
+  * White box is now supported on Linux! ([5075](https://github.com/o3de/o3de/pull/5075))
+ 
+## Deprecations
+ 
+* **Physics**
+    * `TransformUniformScale` ([7573](https://github.com/o3de/o3de/issues/7573))
+    * `BoxManipulatorRequestBus` ([7572](https://github.com/o3de/o3de/issues/7572))
+* **EMFX (Character)**
+    * OpenGL Render Plugin is now deprecated and replaced by EMFX Atom Viewport.
+ 
+## Known issues
+ 
+* **EMFX (Character)**
+    * If you have saved a layout in the old viewport, it will load the old OpenGL viewport. A workaround is to delete the OpenGL widget and add a new Atom viewport widget, then save the layout.  Alternatively, you can recreate the same layout from the default layout.
+ 
+* **Atom (Graphics)**
+    * [Linux] Editor crashes when pressing the **Activate LUT** button right after generating LUT in the HDR Color Grading component. ([9157](https://github.com/o3de/o3de/issues/9157))
+ 
+* **Asset Processor**
+    * [Linux]: Closing Editor while Asset Processor is still running will result in a crash upon relaunching Editor.
+ 
```

Comment:
Adding known issue for Linux per team request.

```suggestion

* In rare situations, on Linux Ubuntu 20.04.4 LTS, it might not be possible to launch O3DE Editor, Project Manager, Asset Processor, or other executables from the file explorer, and an error is displayed.

    **Workaround:** Use the command line to launch the executable from its directory, for example `sudo ./o3de`. Refer to [issue 9502](https://github.com/o3de/o3de/issues/9502) for more infrormation.
```